src/components/templates/Page.jsx

```diff
@@ -110,6 +120,7 @@ export const Page = ({ page, noBacklink }) => {
                 href="https://www.onescreener.com"
                 target="_blank"
                 title="created with onescreener.com"
+                previewMode={mode}
```

Comment:
set flag `isMobilePreview` for children components. Like this you only need to query the `mode === 'MOBILE'` once and the interdependencies between values and UI logic is reduced